Cargo.toml

```diff
@@ -13,10 +13,10 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-libc = "0.2.65"
+libc = "0.2.81"
 alsa-sys = "0.3.0"
```

Comment:
Is there anything bad with existing versions of libc/nix that motivates the dependency update? AFAIK, "0.2.65" means  "0.2.65 or higher".

Reply:
Nah, doesn't matter. I'll revert this part.